### ISSUE TITLE
I can't find what I'm looking for: integrations/angular.md

### ISSUE THREAD

**Issue by michaelwechner** [tinymce/tinymce-angular#231]
Hi

I have integrated TinyMCE 

"tinymce": "^5.7.1",
"@tinymce/tinymce-angular": "^4.2.2",

according to

https://www.tiny.cloud/docs/integrations/angular/

and everything works fine, but unfortunately on Firefox I got an error re  "NS_ERROR_UNEXPECTED"

According to https://github.com/tinymce/tinymce/issues/794 and other posts one should be able to fix this by calling

tinymce.remove('my_tinymce')

assung that "id" is

&lt;editor id="my_tinymce"&gt;...&lt;/editor&gt;

but it is not clear to me how I can call tinymce within the typescript when using Angular.

Thanks for your help in advance!

**Comment by jscasca** [comment]
Hi @michaelwechner you should be able to declare the `Tinymce` at the top of your file and call the variable from there. Here is an exmaple: https://codesandbox.io/s/tinymceangular-forked-i3ix1?file=/src/app/app.component.ts

Let me know if that helps

**Comment by jscasca** [comment]
What version of FF are you using? I have not been able to replicate the issue, unfortunately

**Comment by michaelwechner** [comment]
> Hi @michaelwechner you should be able to declare the `Tinymce` at the top of your file and call the variable from there. Here is an exmaple: https://codesandbox.io/s/tinymceangular-forked-i3ix1?file=/src/app/app.component.ts
> 
> Let me know if that helps

Yes, I was able to reference inside the typescript file as described in the examples above

declare var tinymce: any;

**Comment by michaelwechner** [comment]
> What version of FF are you using? I have not been able to replicate the issue, unfortunately

I am currently using FF 88.0 (64-Bit) on Mac

**Comment by jscasca** [comment]
Just checking if you are still having issues with FF since we could not reproduce the specified behaviour

Is this still the case?

**Comment by michaelwechner** [comment]
Hi

I could fix it with a workaround :-)

Thanks

Michael

Am 02.06.21 um 09:59 schrieb jscasca:
